packages/test/test-version-utils/src/describeCompat.ts

```diff
@@ -20,17 +20,21 @@ import {
 	CompatApis,
 	getDriverApi,
 } from "./testApi.js";
-
 // See doc comment on mochaGlobalSetup.
 await mochaGlobalSetup();
 
+/**
+ * @internal
+ */
+export type CompatVersionKind = "Full" | "NoCompat" | "LoaderCompat" | string;
```

Comment:
Would be nice to have docs here to explain the difference between each `CompatVersionKind`

Reply:
Also why do we need the `| string` here? I imagine if/when we add more compat kinds, we can just add them on the fly.